src/browser/js/contextMenu.js

```diff
@@ -22,12 +22,14 @@ function getSuggestionsMenus(win, suggestions) {
 function getSpellCheckerLocaleMenus(onSelectSpellCheckerLocale) {
   const currentLocale = ipcRenderer.sendSync('get-spellchecker-locale');
   const locales = [
+    {language: 'English (UK)', locale: 'en-GB'},
     {language: 'English (US)', locale: 'en-US'},
     {language: 'French', locale: 'fr-FR'},
     {language: 'German', locale: 'de-DE'},
-    {language: 'Spanish', locale: 'es-ES'},
+    {language: 'Portuguese (BR)', locale: 'pt-BR'},
+    {language: 'Spanish (ES)', locale: 'es-ES'},
```

Comment:
@enahum Is it a correct reference to say `Spanish (ES)` and `Spanish (MX)`?

Because I also see this on Facebook, for instance.

![image](https://user-images.githubusercontent.com/13119842/47046609-8c437a00-d163-11e8-9538-83dfbb4e255d.png)

Reply:
@enahum ping on this ^

Reply:
those are specific to countries, if we are adding support for that cool